File: bijele.cpp
#include <iostream>

int main() {
  int chessset[6] = {1, 1, 2, 2, 2, 8};
  int input;

  for (int i = 0; i < 6; i++) {
    std::cin >> input;
    if (i != 5) {
      std::cout << chessset[i] - input << " ";
    }
    else {
      std::cout << chessset[i] - input << std::endl;
    }
  }

  return 0;
}

File: bijele.py
import sys

chessset = [1, 1, 2, 2, 2, 8]
inputset = input().split()

for i in range(6):
    sys.stdout.write(str(chessset[i] - int(inputset[i])))
    if i != 5:
        sys.stdout.write(' ')
    else:
        sys.stdout.write('\n')
